src/Board.py

```diff
@@ -52,6 +58,18 @@ def __init__(self, style: str = "default", size: int = 5) -> None:
         else:
             raise ValueError("Unrecognised map style")
 
+    def start(self, terrain_filter: Terrain | None = None):
+        return_list = []
+        for row in range(0, len(self.data)):
+            for spot in range(0, len(self.data[row])):
```

Comment:
Spot is a bit ambiguous as a term, perhaps `col` might be good here? (I know it's not exactly a 2D grid but we're treating it like one so I think calling it a column is not too confusing)